tensorflow_io/core/python/ops/version_ops.py

```diff
@@ -14,5 +14,5 @@
 # ==============================================================================
 """version_ops"""
 
-package = "tensorflow>=2.3.0,<2.4.0"
-version = "0.16.0"
+package = "tensorflow>=2.4.0rc0,<2.5.0"
+version = "0.17.0"
```

Comment:
Is the `version==0.17.0` change intentional?

Reply:
@kvignesh1420 Yes we want to bump the version so that it will not interference with docker build (we don't want 0.16's docker image depends on "tensorflow>=2.4.0rc0,<2.5.0" as it will not work (0.16 only works with TF 2.3).